src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,111 @@
+import React, {
+  useCallback, useMemo, useRef, useState,
+} from 'react';
+// eslint-disable-next-line
+import debounce from 'lodash.debounce';
+import { peopleFromServer } from '../data/people';
```

Comment:
You don't need to directly import the data here. Just send it as props to Autocomplete component.